### ISSUE TITLE
Easy way to add TailwindCSS to Remix Project?

### ISSUE THREAD

**Issue by AjayTheWizard** [tailwindlabs/tailwindcss.com#1373]
I found another way to setup TailwindCSS for Remix Project which is so easy to setup rather than working with `concurrently` which makes development experience worse due to large time for compiling CSS Files.

Here the name of package `remix-tailwind`. May be You could introduce in Tailwind Docs or I am edit it and make a pull request to this repo ?

Thanks in advance ;-)

**Comment by reinink** [comment]
Hey! 👋

Thanks so much for this suggestion, but in general we've tried hard to follow the recommended installation instructions provided by the framework's themselves. That's where the use of `concurrently` came from, that's what was recommended by Remix.

While `remix-tailwind` looks neat, we're always careful with recommending new packages like this that don't have much adoption in the community yet. But if it works well for you, great! 💪

That said, just looking at the Remix docs again I can see that they're now actually recommending the use of `npm-run-all`, so if anything it might be worth considering switching to that instead.

https://remix.run/docs/en/v1/guides/styling#tailwind-css

Hope that makes sense!
